shaptools/netweaver.py

```diff
@@ -147,6 +151,32 @@ def is_installed(self, sap_instance=None):
         else:
             raise ValueError('provided sap instance type is not valid: {}'.format(sap_instance))
 
+    @classmethod
+    def update_conf_file(cls, conf_file, **kwargs):
+        """
+        Update NW installation config file parameters. Add the parameters if they don't exist
+
+        Args:
+            conf_file (str): Path to the netweaver installation configuration file
+            kwargs (opt): Dictionary with the values to be updated.
+                Use the exact name of the netweaver configuration file
+
+        kwargs can be used in the next two modes:
+            update_conf_file(conf_file, sid='HA1', hostname='hacert01')
+            update_conf_file(conf_file, **{'sid': 'HA1', 'hostname': 'hacert01'})
+        """
+        for key, value in kwargs.items():
+            pattern = '{key}( *)=.*'.format(key=key)
+            new_value = '{key} = {value}'.format(key=key, value=value)
+            with open(conf_file, 'r+') as file_cache:
+                if key in file_cache.read():
+                    for line in fileinput.input(conf_file, inplace=1):
+                        line = re.sub(pattern, new_value, line)
+                        print(line, end='')
+                else:
+                    file_cache.write('\n'*5+new_value)
```

Comment:
What is this `5+` thing?? It is just used to add 5 empty lines? Do we need this?

Reply:
yes `'\n'*5` is for the empty lines. I kept it as such to match the existing file structure. We don't really need it, I can remove it